Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -965,23 +965,6 @@ public function __construct()
         $this->makeTest($expected);
     }
 
-    public function testStatic()
-    {
-        $expected = <<<'EOF'
-<?php
-
-static class Foo
-{
-    static function Foo()
```

Comment:
Imo the test case should be removed. it is not valid, result is a fatal error:

https://3v4l.org/CTJV0